src/schemas/json/tslint.json

```
@@ -2067,6 +2084,23 @@
             }
           ]
         },
+        "no-null-undefined-union": {
+          "description": "Disallows explicitly declared or implicitly returned union types with both null and undefined as members.",
```

```suggestion
          "description": "Forbid explicitly declared or implicitly returned union types with both null and undefined as members\nhttps://palantir.github.io/tslint/rules/no-null-undefined-union/",
```

The same issue as above.